tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -59,324 +63,321 @@ create_boundary_condition() {
       Direction<3>::upper_zeta(), 50);
 }
 
-auto make_domain_creator(const std::string& opt_string,
-                         const bool use_boundary_conditions) {
-  if (use_boundary_conditions) {
-    return TestHelpers::test_option_tag<
-        domain::OptionTags::DomainCreator<3>,
-        TestHelpers::domain::BoundaryConditions::
-            MetavariablesWithBoundaryConditions<3, domain::creators::Sphere>>(
-        opt_string + std::string{"  BoundaryCondition:\n"
-                                 "    TestBoundaryCondition:\n"
-                                 "      Direction: upper-zeta\n"
-                                 "      BlockId: 50\n"});
+// Calculate block logical coordinates
+tnsr::I<double, 3, Frame::BlockLogical> logical_coords(
+    std::mt19937 gen, const size_t num_blocks, const size_t block_id,
```

Comment:
Take the generator by reference I think: `const gsl::not_null<std::mt19937*> generator`